cdap-ui/app/features/adapters/controllers/detail-ctrl.js

```diff
@@ -35,7 +35,9 @@ angular.module(PKG.name + '.feature.adapters')
     dataSrc.poll({
       _cdapNsPath: path + '/status'
     }, function(res) {
-      $scope.status = res.status;
+      // Comment it out https://issues.cask.co/browse/CDAP-2346 when fixed.
```

Comment:
Can you undo this?
There's this PR that should resolve this.
https://github.com/caskdata/cdap/pull/2405